ocw-course-v2/ocw-studio.yaml

```
@@ -109,6 +109,67 @@ collections:
           filter_type: "equals"
           value: "Video"
 
+  - category: Content
+    name: link
+    label: Links
+    folder: content/links
+    fields:
+      - label: Title
+        name: title
+        widget: string
+        required: true
+
+      - label: Text
+        name: text
+        widget: string
+        required: true
+        help: Text to display as the link.
+
+      - label: URL
+        name: external_url
+        widget: string
+        required: true
+        help: The URL. For example, https://ocw.mit.edu/
+
+      - label: License
+        name: license
+        widget: select
+        options:
+          - label: CC-BY-NC-SA
```

There shouldn't be a dash between the CC and the rest of the string. This is an error in our existing project & data. 

```suggestion
          - label: CC BY-NC-SA
```

See https://creativecommons.org/licenses/by-nc-sa/4.0/deed.en